lib/omnitrade_client.dart

```diff
@@ -0,0 +1,24 @@
+library omnitrade_client;
+
+import 'package:chopper/chopper.dart';
+import 'src/services.dart';
+import 'package:http/http.dart' as http;
+
+class OmniCredentials {}
+
+/// A OmniClient.
+class OmniTradeClient extends OmniServices {
+  OmniTradeClient({this.httpClient, this.credentials}) {
+    credentials ??= OmniCredentials();
```

Comment:
Do we need these in constructors? In what case would an instance variable would not be null inside a constructor?